changelog.json

```
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "3.2.0": [
+      "[New] Preview your pull request - view a diff of changes in your current branch.",
+      "[Added] Add Tabby terminal integration for macOS - #16040. Thanks @ansidev!",
+      "[Added] Add JetBrains DataSpell support on Windows - #16020. Thanks @tsvetilian-ty!",
+      "[Added] Add JetBrains DataSpell support on macOS- #16020. Thanks @jm-rivera!",
```

Should we combine these two?
```suggestion
      "[Added] Add JetBrains DataSpell support - #16020. Thanks @tsvetilian-ty and @jm-rivera!",
```